tools/goreleaser-config/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/smartcontractkit/chainlink/tools/goreleaser-config
 
-go 1.23.0
+go 1.22.8
```

Comment:
Is this intentional?

Reply:
i want to say yes, but i do not remember the reasoning behind it now :/